a_test.go

```diff
@@ -1,10 +0,0 @@
-package holochain
-
-import (
-	"testing"
-)
-
-// needed to setup the holochain environment, not really a test.
-func Test(t *testing.T) {
-	Initialize()
-}
```

Comment:
I was having trouble running single tests with --run and found out about TestMain, see added code below.